### PR TITLE
More restructuring in preparation for 0.8

### PR DESCRIPTION
**Pull request by oicheryl**

### CODE REVIEW

File: _docs/install/docker/index.md [hunk 1]
```diff
@@ -2,64 +2,26 @@
...
-### Routable IP Address
-
-StorageOS nodes must be able to contact each other over the network.  By default,
-the node's first non-loopback address will be configured as the `ADVERTISE_IP`.
-In some cases (such as with Vagrant installations), this will not be appropriate
-and it will need to be set manually.
-
-Use `ip a` to list available ip addresses, and then configure StorageOS to use a
-specific address by adding `-e ADVERTISE_IP=<ip>` to the StorageOS docker run
-command.
```
Comment (croomes) [inline _docs/install/docker/index.md]: Not specifying the ADVERTISE_IP has been a common mistake - can we highlight the requirement more?

File: _docs/introduction/quick_start.html [hunk 2]
```diff
@@ -0,0 +1 @@
+<meta http-equiv="refresh" content="0; url=http://localhost:4000/docs/install/quickstart" />
```
Comment (croomes) [inline _docs/introduction/quick_start.html]: Don't think you meant to include this?

File: _docs/reference/open_source_attribution.md [hunk 3]
```diff
@@ -586,32 +586,6 @@ limitations under the License.
...
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
-FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
-COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
-IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
-CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
-```
-
-
```
Comment (croomes) [inline _docs/reference/open_source_attribution.md]: We still import boltdb as a library (etcd uses it)

File: _docs/introduction/quick_start.html (hunk 2 above)
Comment (oicheryl) [inline _docs/introduction/quick_start.html]: I did - there's a few links that still point to the this page, so Chris asked for a redirect to the right quickstart.

File: _docs/reference/open_source_attribution.md (hunk 3 above)
Comment (oicheryl) [inline _docs/reference/open_source_attribution.md]: Done (reverted this file).

File: _docs/install/docker/index.md (hunk 1 above)
Comment (oicheryl) [inline _docs/install/docker/index.md]: I've highlighted it in install/docker/container and install/docker/plugin.

**Comment by oicheryl** [comment]
Ready for rereview @croomes.

File: _docs/introduction/quick_start.html (hunk 2 above)
Comment (croomes) [inline _docs/introduction/quick_start.html]: You probably want to redirect to http://docs.storageos.com/docs/install/quickstart

Comment (oicheryl) [inline _docs/introduction/quick_start.html]: Oops.

**Pull request merged**